src/gis-utils/components/ol-geolocate.ts

```diff
@@ -80,4 +82,21 @@ export class Geolocate extends ol.control.Control {
     );
     return feature;
   }
+
+  private _addPositionFeature(pos, view, source) {
+    const zoomLevel = this.options.zoomLevel ? this.options.zoomLevel : 12;
+
+    const positionFeature = this._createFeature();
+    const coordinates = ol.proj.transform(
+      [pos.coords.longitude, pos.coords.latitude],
+      'EPSG:4326',
+      view.getProjection()
+    )
+    view.setCenter(coordinates);
+    view.setZoom(zoomLevel);
```

Comment:
hier ben ik niet zeker of we dit willen doen bij elke update, als de gebuiker ondertussen zelf zoomt of ergens anders heen pant, komt hij altijd terug op deze extent terecht, wat vervelend kan zijn. Dus indien mogelijk 1x zoomen & cetreren, nadien enkel nog de lat lon updaten.